confidential-data-hub/hub/src/config.rs

```diff
@@ -41,19 +41,82 @@ impl KbsConfig {
     }
 }
 
-#[derive(Deserialize, Debug, PartialEq)]
+#[derive(Clone, Deserialize, Debug, PartialEq)]
 pub struct Credential {
     pub resource_uri: String,
     pub path: String,
 }
 
-#[derive(Deserialize, Debug, PartialEq)]
+#[derive(Clone, Deserialize, Debug, PartialEq)]
+pub struct ImageConfiguration {
+    /// Whether to enable image security validation
+    ///
+    /// This defaults to `false`.
+    pub security_validate: bool,
+
+    /// Use `auth.json` control
+    /// ///
+    /// This defaults to `false`.
+    pub auth: bool,
```

Comment:
Optionally idea: Maybe this is an opportunity to make this field clearer, which something like `use_credentials`, or `authenticated_registry_support` rather then just `auth`?

Reply:
Nice suggestion.

Reply:
Is `use_registry_credential` a good name?

Reply:
Maybe `use_registry_credentials` as IIRC you can specify multiple credentials in a single auth.json file?